client/extensions/woocommerce/app/order/order-created.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'gridicons';
+import { localize, moment } from 'i18n-calypso';
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/16777/commits/6eb94069be9491071840c11c9e1607062f0d1861